docs/portal.properties-Reference.md

```diff
@@ -131,3 +131,16 @@ To change the gene sets used for gene querying, create a JSON file and add gene
 ```
 querypage.setsofgenes.location=file:/<path>
 ```
+# Reference Genome information for importing gene and study
```

Comment:
6. From a user perspective, it would make sense to just give one type of build name to add it to his configuration, instead of using 3 overlapping properties. Ideally something like this should be enough:
```genome.build= GRCh37; GRCh38```

Can we get the link Human <- hg19 <- GRCh37 from somewhere else?

Reply:
@sandertan ,
thanks for the suggestions. Here is modified version. Please take  a look see if anything missed. 
>
# Reference Genome information for validating the genomic data
The reference genome information will be used by the importer validation script to ensure the reference genome version matching the reference genome version used to analyse the genomic data. 
Further details on the reference genome can be found in [this document](import-reference-genome.md); details on how to load reference genes can be found in [this document](Updating-gene-and-gene_alias-tables.md)

1. use **species** to indicate type of organisms of the reference genome, for example, **human** for homo species.
2. use **genome.name** to name the reference genome. In theory you can give any names to a reference genome. By convention, the name of reference genome always has a prefix of **hg** for human, for mouse a prefix of **mm**.
3. use **genome.build** to specify different versions of sequence database assembled by different publishers for a given reference genome. For example, 
a genome assembly published by NCBI always has a prefix of **GRCh** for homo species. 

```
# species and genomic information
species=human
genome.name=hg19,hg19,hg38
genome.build=37,GRCh37,GRCh38
```

Reply:
@sandertan , to question 6:
the order those three properties does not matter, but one genome could have multiple builds. In the script three properties will be parsed and stored in a dictionary:
```
{'human':{'hg19':['37','GRCh37']},{'hg38':['GRCh38']}}
```